generate_testdata.sh

```diff
@@ -79,6 +79,10 @@ scaffold_test_project() {
             $kb create api --group sea-creatures --version v1beta1 --kind Kraken --controller=true --resource=true --make=false
             $kb create api --group sea-creatures --version v1beta2 --kind Leviathan --controller=true --resource=true --make=false
             $kb create api --group foo.policy --version v1 --kind HealthCheckPolicy --controller=true --resource=true --make=false
+            if [ $project == "project-v3-multigroup" ]; then
+              $kb create api --version v1 --kind Lakers --controller=true --resource=true --make=false
+              $kb create webhook --version v1 --kind Lakers --defaulting --programmatic-validation
+            fi
           elif [ $project == "project-v2-addon" ] || [ $project == "project-v3-addon" ]; then
             header_text 'enabling --pattern flag ...'
             export KUBEBUILDER_ENABLE_PLUGINS=1
```

Comment:
We need to the same for project-v3 and project-v3-addon too.

Reply:
@camilamacedo86 I need to create a separate project for those "project-v3-emptygroup" and "project-v3-addon-emptygroup" because multiple groups are not enabled by default. Let me know if we should that?

Reply:
Done

Reply:
Sorry, it was my mistake. It is true we cannot add this scenario in the other both projects because they do not support multigroup. we should not add new scenarios because will be hard to keep to maintain many of them.

Reply:
Reverted.

Reply:
You can close this one.